### PR TITLE
Update Drafts Page based on New Design

### LINKED ISSUE #1150: Drafts page lifting: add last update time, remove "this draft" from "Delete this draft button, align button to right
Update Draft Page UI based on Paresh's design

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1150

Changes:
- Styling updates and add some html elements to draft row for separation mostly
- Update styles based on design, padding around should be 14px, similar to mockup

screen shot 2017-12-04 at 8 11 02 am

### CODE REVIEW

**Comment by Sekhmet** [comment]
Looks great!
I think that Delete button should be centred in the middle:
image

**Comment by jm90m** [comment]
@Sekhmet  centered in middle of the page? or text centered with lastUpdated?

screen shot 2017-12-04 at 9 28 50 am

or centered between title and last updated text?

**Comment by bonustrack** [comment]
Can we have the title in bold aswell?

File: src/client/locales/en.json [hunk 1]
```diff
@@ -182,10 +182,12 @@
   "get_boost_content": "Want to boost your post? Add the topic \"busy\" and get a like from us.",
   "drafts": "Drafts",
   "drafts_empty": "You don't have any draft saved",
+  "drafts_description": "Posts that never made public. You can republish them or delete them.",
```
Comment (bonustrack) [inline src/client/locales/en.json]: What you mean by republish them? We should find a better sentence, it feel little bit confusing.

Comment (jm90m) [reply src/client/locales/en.json]: I just made it exactly how it was on the mockup lol https://cdn.discordapp.com/attachments/371597897055076363/382511581055025152/Drafts.png

**Comment by Sekhmet** [comment]
@jm90m on the design `Delete` is centred in the middle of DraftRow. On the website it's centred inside bottom half of it. It looks kind of weird to me.

**Comment by jm90m** [comment]
@Sekhmet ohhh I see what you mean now

File: src/client/locales/en.json (hunk 1 above)
Comment (jm90m) [reply src/client/locales/en.json]: updating the text now to `These are posts that were never made public. You can publish them or delete them.`

**Comment by bonustrack** [comment]
Maybe when there is no date we can hide totally the "Last updated Invalid date":
image
Also some post not seem to be sorted by date.

**Comment by mynameisek** [comment]
I think when we click on a Draft post, for now it's opening the Editor page, with a right sidebar. This is not a good  UX in my opinion.
My suggestion:
- Keep the sidebar left (Draft page) (Activity/Bookmarks/Drafts/Edit profile/Invite/Settings)
- When click on a Draft post,  the editor should directly shows up in the same layout so that the user can click on the left sidebar menu (Draft link) to go back to Draft list.
do you agree @jm90m

**Comment by jm90m** [comment]
@bonustrack just updated it not display invalid dates, i can't reproduce the order issue though.. hmm..

**Comment by jm90m** [comment]
@mynameisek yeah I think sidebar will look good when clicking on draft. I'll see if I can add it in

**Comment by jm90m** [comment]
@mynameisek added sidebar only for draft pages, check it out and see if you like it

**Comment by mynameisek** [comment]
@jm90m issue with this layout is that it doesnt display the post (preview) like it will appear once published.. So maybe to remove right sidebar ? what do you think. OR :

The other solution is the following:
- Keep it as before: Mean Editor + Right sidebar, 
- but to solve the UX issue, add in the Rightsidebar a "DRAFT" Widgets, with all your drafts list and a link to display all. (back to /drafts)

**Comment by jm90m** [comment]
@Sekhmet Since @bonustrack is away for a few days, whenever you got time, could you do a review on this? :) thanks!

**Comment by bonustrack** [comment]
When i have a window around 920px width the margin in left and right are not egal:
image
Can we fix this?

**Comment by jm90m** [comment]
@bonustrack yeah will look into it

**Comment by jm90m** [comment]
@bonustrack just updated it so draft page has its own layout styles, will probably be useful in future when we add a sidebar to drafts page

**Comment by bonustrack** [comment]
Just figured out that the delete button is not accessible on mobile screen:
image
We should have a different style on mobile, maybe show the button by default, with clear both

**Comment by Sekhmet** [comment]
@pareshUD do we have mobile design for this?

**Comment by jm90m** [comment]
@bonustrack ahh i see what you mean, yeah I'll update it to always be shown for screen sizes under 998px 
```
@medium: ~"only screen and (min-width: 998px)";
```

**Comment by bonustrack** [comment]
The layout should be constant, now the button delete appear in left or bottom depending on the size of the title:
image
Also on the first draft in the screenshot the title overflow the block padding, it should not appear like this.

**Comment by jm90m** [comment]
thats because it uses flex box to flex-wrap @bonustrack

**Pull request merged**